SVGLayer.h

```diff
@@ -8,12 +8,76 @@
 
 #import <QuartzCore/QuartzCore.h>
 
-/// An SVG-backed CALayer that you can view that parses, renders, and display an SVG file.
+/// A CALayer subclass that renders an SVG file.
 
 @interface SVGLayer : CALayer
+
+NS_ASSUME_NONNULL_BEGIN
+
+/*!
+ *  @discussion Renders the provided SVG.
+ *
+ *  @param aSVGName The name of an SVG file in the app bundle.
+ *
+ *  @code // iOS:
+ let myLayer = SVGLayer()
+ myLayer.frame = CGRect(x: 0, y: 0, width: 100, height: 100)
+ myLayer.renderSVGNamed("tiger")
+ view.layer.addSublayer(myLayer)
+
+ // macOS:
+ let myLayer = SVGLayer()
+ myLayer.frame = CGRect(x: 0, y: 0, width: 100, height: 100)
+ myLayer.geometryFlipped = true
+ myLayer.renderSVGNamed("tiger")
+ view.layer?.addSublayer(myLayer)
+ *
+ */
+- (void)renderSVGNamed:(NSString *)aName;
```

Comment:
I see. 

But I think that having this as a _method_ is not very consistent with the rest of the API. Currently, if I want to load an SVG into an SVGLayer or SVGImageView then I have to call a method _if I know the name of the file in the bundle_, otherwise I need to update a variable _if I have its raw XML_. You should be able to load an SVG in a consistent way regardless of the form it takes (file path or XML).

I think we should offer these three different initialisers for SVGLayer and SVGImageView:

`- (instancetype)initWithData:(NSData *)data`
`- (instancetype)initWithSVGSource:(NSString *)svgSource`
`- (instancetype)initWithContentsOfFile:(NSString *)path`
`- (instancetype)initWithContentsOfURL:(NSURL *)url;`

So that the layer/imageview is always properly initialised. 

To be consistent with the `svgSource` property,  the `loadSVGNamed` can be replaced by this property:

`@property(nonatomic, copy) NSString *svgName;`

Any thoughts?